packages/viewer/src/components/Properties/List.svelte

```diff
@@ -21,7 +21,7 @@
   {#each $properties as property (property.id)}
     <tr>
       <td>
-        <Link.Property {property}>{property.id}</Link.Property>
+        <Link.Property {property}>P{property.id}</Link.Property>
```

Comment:
Any particular reason you opted for the form _without_ extra `0`s here? (Same question throughout the PR.)

Reply:
In any case, I'm weakly inclined to update the `Link.*` components to take an optional `content: "id" | "name" = "name"` field and have _it_ do the id formatting (in large part – so that if we decide to change this, we have one place to do so).

Reply:
> Any particular reason you opted for the form without extra `0`s here?

Purely out of laziness; I think padding 0s would be better.

Reply:
well hang on. I do like the short IDs in the table view. (Currently we just show e.g. `34` but would like `P34`. Here `P000034` is taking up too much real estate.) So I think having `<Link/>` be responsible here makes sense.

Reply:
I implemented this in [cf5acc3](https://github.com/pi-base/web/pull/88/commits/cf5acc31361bb9bceab23beb84d6b0add0ce43a1)